playground/app.js

```diff
@@ -89,6 +99,7 @@ async function buildVueApp() {
     {
       pipeline: initialPipeline,
       currentDomain: 'test-collection',
+      pagesize: 2,
```

Comment:
We can remove this row to fall back to the default `pagesize` (`50` as of today) ?